packages/core/src/routes/interaction/utils/sign-in-experience-validation.ts

```diff
@@ -59,7 +59,7 @@ export const identifierValidation = (
         if (
           'passcode' in identifier &&
           !verificationCode &&
-          ![SignUpIdentifier.Email, SignUpIdentifier.EmailOrSms].includes(signUp.identifier) &&
+          !signUp.identifiers.includes(SignInIdentifier.Email) &&
```

Comment:
Why check if signUp identifiers include a `sign in` identifier? I'm really confused